dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/handlers/AndroidLogsHandler.kt

```diff
@@ -0,0 +1,50 @@
+package com.datadog.android.tracing.internal.handlers
+
+import android.util.Log
+import com.datadog.android.log.Logger
+import datadog.opentracing.LogsHandler
+import java.util.concurrent.TimeUnit
+
+internal class AndroidLogsHandler(
```

Comment:
Maybe you could rename this class `AndroidSpanLogsHandler` ? Cause everytime I come upon it I think it's for the logs feature and not spans